application.js

```diff
@@ -7,12 +7,43 @@ var bodyParser = require('body-parser');
 var raincatcherUser = require('fh-wfm-user/lib/router/mbaas');
 var sessionInit = require('./lib/sessionInit');
 var adminRouter = require('./lib/routes/admin');
+var fhComponentMetrics = require('fh-component-metrics');
 
 // list the endpoints which you want to make securable here
 var securableEndpoints;
 
 var app = express();
 
+function metricsEnabled(metricsCfg) {
+  return metricsCfg.enabled === 'true' && metricsCfg.host && metricsCfg.port;
+}
+
+var metricsConf = {
+  host: process.env.METRICS_HOST,
+  port: process.env.METRICS_PORT,
+  title: 'auth-svc-' + process.env.FH_TITLE,
+  enabled: process.env.METRICS_ENABLED || 'false',
+  cpu_interval: process.env.METRICS_CPU_INTERVAL |0 || 2000,
+  mem_interval: process.env.METRICS_MEM_INTERVAL |0 || 1000
```

Comment:
@paolobueno yes, that's the intention (more specifically to always return an int). I'll change it to `parseInt()` (`Number()` could give non-int numbers (which would probably be fine in this case, but I don't want to have to test it :P), and `_.toNumber` means adding a dependency to this file for just this).

Although `|0` can be considered a hack, it has some good properties also:

* when you always want to return an int, even for bad input it will return 0 (`parseInt` will return `NaN`)
* If you want to turn a boolean into a meaningful int (`true|0` will give 1, whereas `parseInt(true)` will give NaN)

With the disadvantage that it only works for 32-bit integers (for very large numbers it will give unexpected results.